src/backend/postmaster/pgstat.c

```diff
@@ -2953,6 +2953,8 @@ pgstat_report_resgroup(TimestampTz queueStart, Oid groupid)
 		beentry->st_waiting = PGBE_WAITING_RESGROUP;
 	}
 
+	elog(DEBUG1, "Changed resource groupId to %d", groupid);
```

Comment:
This function seems to be about updating statistics, not about changing the resource group.